app/views/course/condition/achievements/edit.html.slim

```diff
@@ -0,0 +1,5 @@
+h2= t('.header')
+
+== render 'form'
```

Comment:
= doesn't escape HTML because Rails has another method called `html_safe?` I rather everyone use = than risk forgetting to escape something and introduce an injection attack.